healthScore/templates/login.html

```diff
@@ -25,8 +108,16 @@ <h2>Login</h2>
                 <label for="password">Password:</label>
                 <input type="password" class="form-control" id="password" name="password" placeholder="Password">
             </div>
+            <div style="text-align:left; margin-top:20px;">
```

Comment:
Could you please add the styles in a class?

Reply:
Updated